airflow/www/templates/airflow/version.html

```diff
@@ -0,0 +1,34 @@
+{#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#}
+{% extends "airflow/master.html" %}
+
+{% block body %}
+    {{ super() }}
+    <h2>{{ title }}</h2>
+    {% if airflow_version %}
+        <h4>Version : <a href="https://pypi.python.org/pypi/airflow/{{ airflow_version }}"> {{ airflow_version }}</a></h4>
+    {% else %}
+        <h4>Version : Not Available</a></h4>
+    {% endif %}
+
+    {% if git_version %}
```

Comment:
Nit: use same title for both of these (git version vs git commit tree), can also refactor this to avoid duplication (the only difference being "{{ git_version }}" vs "Not Available"